src/index.html

```diff
@@ -13,5 +13,178 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset_form">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input 
+          type="text"
+          name="surname"
+          class="field"
+          autocomplete="off"
+          required
```

Comment:
```suggestion
            type="text"
            name="surname"
            class="field"
            autocomplete="off"
            required
```

add 2 spaces before attributes and fix it **everywhere** in the document

Reply:
fixed